tests/SyslogTest.php

```diff
@@ -23,6 +23,19 @@ private function createData($line, $resultDelta) {
         return $data;
     }
 
+
+    /**
+     * Test an input line with the modified fields
+     *
+     * @param string $inputline The line from the syslog daemon including the ||'s
+     * @param array $modified of the modified fields, most likely containging the keys program and msg
+     */
+    private function checkSyslog($inputline, $modified) {
```

Comment:
Would you mind just updating this function to use just 4 spaces for indentation.